ko/template_extending/README.md

```diff
@@ -139,6 +139,6 @@
 {% endblock %}
 ```
 
-다 되었어요! 여러분의 웹사이트가 잘 작동하는지 확인해보세요.)
+다 되었어요! 여러분의 웹사이트가 잘 작동하는지 확인해보세요. :)
```

Comment:
[Fix Typo] 
Original Text : `That's it! Save the file, and check if your website is still working properly. :)`